root/report/releases_without_disc_ids.tt

```diff
@@ -0,0 +1,16 @@
+[%- WRAPPER 'layout.tt' title=l('Releases Without Disc IDs') full_width=1 -%]
+
+<h1>[% l('Releases Without Disc IDs') %]</h1>
+
+<ul>
+  <li>[% l('This report shows releases (official and promotional only) that have at least
+            one medium without a Disc ID. For instructions on how to add one, see the {add_discids|documentation page}.',
```

Comment:
The same remark as for the title. Maybe a bit more detailed explanation would clarify it.